deps.edn

```diff
@@ -38,6 +38,7 @@
   com.draines/postal                        {:mvn/version "2.0.5"}              ; SMTP library
   ; Detect the charset in uploaded CSV files
   com.github.albfernandez/juniversalchardet {:mvn/version "2.5.0"}
+  com.github.jknack/handlebars              {:mvn/version "4.3.1"}
```

Comment:
can't use 4.4 because it supports java 17+

Reply:
Let's put that comment in the code, pertinent for maintenance

Reply:
do it like `^:antq/exclude ; 4.4 requires java 17+`